iosHyperskillApp/iosHyperskillApp/Sources/Modules/Home/Views/HomeView.swift

```diff
@@ -87,6 +87,11 @@ struct HomeView: View {
                         )
                     }
 
+                    TopicToDiscoverNextCardView(
+                        state: viewModel.topicsToDiscoverNextStateSk,
+                        delegate: viewModel
+                    )
```

Comment:
Here we should add an extra `16` to the top padding.

See screenshot from Figma (this card is like the start of another section -> bigger padding to show this visually)

<img width="544" alt="Screenshot 2023-02-07 at 3 25 52 PM" src="https://user-images.githubusercontent.com/8586063/217191691-c8477ed2-25d7-40aa-867e-f8e00ba8e83f.png">